### ISSUE TITLE
Stable updates push based on update id

### ISSUE THREAD

**Issue by mohanboddu** [fedora-infra/bodhi#3160]
During release freezes, Fedora QA will request Fedora RelEng to push some of the updates to push to stable if they fix blockers or FE's. For that we use

```
bodhi-push --builds 'nvr,nvr,...'
```

This is okay if the updates have only one build, but it becomes tiresome if there a multiple updates like [this update](https://bodhi.fedoraproject.org/updates/FEDORA-2019-ac2a21ff07) for example.

Can you make it to pass the update id's to push to stable? Something like

```
bodhi-push --updates 'update_id1,update_id2...'
```

**Comment by bowlofeggs** [comment]
Sure, but one note: I think you can put any one of the builds from the update in and I think it'll find it. (Not sure, I haven't tried that, but I'm guessing it'd work.)

**Comment by mohanboddu** [comment]
I will give it a shot next time and I will let you know.

**Comment by sebwoj** [comment]
This unittest shows that update with provided build via cli is locked after bodhi-push:
https://github.com/fedora-infra/bodhi/blob/b7e9a87b474fc7d008164a9e0df41963106abb87/bodhi/tests/server/test_push.py#L420

**Comment by mohanboddu** [comment]
> Sure, but one note: I think you can put any one of the builds from the update in and I think it'll find it. (Not sure, I haven't tried that, but I'm guessing it'd work.)

This is working, so, this ticket is not critical, but good to have.

**Comment by bowlofeggs** [comment]
Fixed in #3205.
